### DISCUSSION TITLE
Looking for Assistance with => Error: Fallback mode unable to write file changes -

### DISCUSSION THREAD

**Discussion by jthiesse-mayoclinic** [jlongster/absurd-sql#40] [Q&A]
First - @jlongster great job! Absurd fits right into a gap and should solve a tricky performance issue for us.

The following, "Error: Fallback mode unable to write file changes", is seen when doing a read or write operation against the database.

Worth noting, the Indexed DB is named 'undefined' rather than the database name specified.

When I clone this repo and run the examples, everything works as expected.

When I copy the https://github.com/jlongster/absurd-example-project/blob/master/src/index.worker.js into my project I get the error.

I am assuming it has to do with the following:

I am currently using create-react-app and using [workerize-loader](https://github.com/developit/workerize-loader) for worker related processing (which has been a success up to this point).

Is anyone aware of how or why using workerize-loader would cause the above error? Or even if workerize is the likely issue, would there be other defaults in create-react-app which may cause issues?

Thank you in advance for the time and help.

Edit: Here is a repo to see the behavior

https://github.com/jthiesse-mayoclinic/absurd-worker

**Comment by quolpr** [comment]
Hey @jthiesse-mayoclinic! the `undefined` issue was fixed at https://github.com/jlongster/absurd-sql/pull/21 , but has not been released yet

**Reply by jthiesse-mayoclinic** [reply]
@quolpr Thanks for the reply. I am seeing the issue in Firefox and Chrome, have not tested in Safari yet. Any thoughts on what would cause the issue in these browsers?

**Reply by quolpr** [reply]
It looks like a CORS issue, https://github.com/jlongster/absurd-sql/issues/14#issuecomment-902116794

**Reply by jthiesse-mayoclinic** [reply]
I attempted to use react-app-rewired to override the webpack config.

New branch:  https://github.com/jthiesse-mayoclinic/absurd-worker/tree/rewire2

```JavaScript
devServer: function(configFn) {
  return function(proxy, allowedHost){
      const config = configFn(proxy, allowedHost);
  
      config.headers= {
          'Cross-Origin-Opener-Policy': 'same-origin',
          'Cross-Origin-Embedder-Policy': 'require-corp'
      };           
  
      return config;
  }
  }
```

Now, we do not get past the database creation line:

```JavaScript
try {
    db = new SQL.Database('/sql/db.sqlite', { filename: true });
  } catch (e) {
    console.error('Error while creating the database', e);
  }
```

No errors.

Ideas on how to debug this?
